cli/src/processSourceFiles.ts

```diff
@@ -188,8 +260,8 @@ export const processSourceFiles = async ({
           paddedAppVersion,
         );
 
-        if (newJsContent === jsContent || newMapContent === mapContent) {
-          console.debug(`Template App version not found in ${jsFilePath}`);
+        if (newJsContent === jsContent && newMapContent === mapContent) {
```

Comment:
@pablomatiasgomez Does this look right? If one file changes, it's technically "found"

Reply:
yep, this looks right, I wonder if this was affecting someone as I wouldn't expect the map content to change when this is replaced?